Makefile

```diff
@@ -213,6 +213,13 @@ merged.lcov:
 	lcov --add-tracefile coverage/unittest.info --add-tracefile coverage/integrationtest.info -a coverage/lcov.info -o merged.lcov
 
 ##@ Utilities
+tls-files:
+ifeq (, $(shell which mkcert))
+ $(error "mkcert is not installed, consider following this instructions: https://github.com/FiloSottile/mkcert#installation ")
```

Comment:
Does there need to be a `dependencies.toml` or a README entry to have devs install this tool?

Reply:
I tried to add a toml entry but this dependency doesn't have the specific binary for x86_64 architectures. 
```
Downloading https://github.com/FiloSottile/mkcert/releases/download/v1.4.3/mkcert-v1.4.3-darwin-x86_64
curl: (22) The requested URL returned error: 404
```
<img width="897" alt="image" src="https://user-images.githubusercontent.com/3892786/157112909-410e202c-546f-432f-a262-c0d45280addd.png">